pkg/deployments/tasks/20230403-timelock-authorizer/input/goerli.ts

```diff
@@ -0,0 +1,619 @@
+import { DAY } from '@balancer-labs/v2-helpers/src/time';
+import Task, { TaskMode } from '../../../src/task';
+import { DelayData, RoleData } from './types';
+import { ANY_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+
+const EVERYWHERE = ANY_ADDRESS;
+
+const Vault = new Task('20210418-vault', TaskMode.READ_ONLY, 'goerli');
+
+const BalancerTokenAdmin = new Task('20220325-balancer-token-admin', TaskMode.READ_ONLY, 'goerli');
+const GaugeController = new Task('20220325-gauge-controller', TaskMode.READ_ONLY, 'goerli');
+const VotingEscrowDelegationProxy = new Task('20220325-ve-delegation', TaskMode.READ_ONLY, 'goerli');
+const SmartWalletChecker = new Task('20220420-smart-wallet-checker', TaskMode.READ_ONLY, 'goerli');
+
+const DAO_MULTISIG = '0x171C0fF5943CE5f133130436A29bF61E26516003';
+
+// Start: block that contains the transaction that deployed the `TimelockAuthorizer`.
+// etherscan.io/tx/0x20eb23f4393fd592240ec788f44fb9658cc6ef487b88398e9b76c910294c4eae
+// End: close to the current block at the time the `TimelockAuthorizerMigrator` is deployed.
+// It is expected that no roles were granted to the old authorizer after it.
+export const TRANSITION_START_BLOCK = 8745337;
+export const TRANSITION_END_BLOCK = 8745437;
```

Comment:
I'm actually not sure we need this now? We used to require the transition fix because the migration took ~1 month, but now that the migration only takes a couple days we don't need to have a transition period. We can just use the permissions hardcoded here and that's it.

Reply:
We don't. You are right. That was part of the design a copied.